### PR TITLE
Add `related merge requests` and `closed by` in Issues service

### LINKED ISSUE #897: Add support to get related related MRs and MRs that close an issue
**Description**
The `Issues` service does not propose the following routes :
https://docs.gitlab.com/ce/api/issues.html#list-merge-requests-related-to-issue
https://docs.gitlab.com/ce/api/issues.html#list-merge-requests-that-will-close-issue-on-merge

**Proposal**
Add support for both, which should be quite straightforward. I can propose a contribution if you want.

### PR DESCRIPTION
**Pull request by fbridault**
This adds two new functions, one to get the related merge requests of an issue, and the other to get the merge requests that close an issue:
- https://docs.gitlab.com/ce/api/issues.html#list-merge-requests-related-to-issue
- https://docs.gitlab.com/ce/api/issues.html#list-merge-requests-that-will-close-issue-on-merge

Doing so, I also sorted two other functions, because the alphabetical order was not respected in a previous commit.

Close #897

### CODE REVIEW

File: packages/gitbeaker-core/src/services/Issues.ts [hunk 1]
```diff
@@ -47,16 +47,16 @@ export class Issues extends BaseService {
...
+  closedBy(projectId: string | number, issueId: number, options?: BaseRequestOptions) {
     const [pId, iId] = [projectId, issueId].map(encodeURIComponent);
 
-    return RequestHelper.put(this, `projects/${pId}/issues/${iId}`, options);
+    return RequestHelper.get(this, `projects/${pId}/issues/${iId}/closed_by`, options);
   }
 
-  links(projectId: string | number, issueIId: number) {
-    const [pId, iId] = [projectId, issueIId].map(encodeURIComponent);
+  edit(projectId: string | number, issueId: number, options?: BaseRequestOptions) {
```
Comment (jdalrymple) [inline packages/gitbeaker-core/src/services/Issues.ts:56]: Though you didnt change this, probably worth fixing in this commit.

Could you rename issueId to [issueIid](https://docs.gitlab.com/ee/api/issues.html#edit-issue)

Comment (jdalrymple) [reply packages/gitbeaker-core/src/services/Issues.ts:56]: Same change for all the new functions too! Just double checked and they all point to the IId and not the Id

Comment (fbridault) [reply packages/gitbeaker-core/src/services/Issues.ts:56]: Yeah I noticed it was not consistent and I considered to do it, but did not want to override my prerogatives. ;-)
If that helps, I'm gonna do it.

Comment (jdalrymple) [reply packages/gitbeaker-core/src/services/Issues.ts]: Thank you!

**Comment by jdalrymple** [comment]
:rocket: PR was released in [`22.0.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/22.0.1) :rocket:

**Pull request merged**